components/brave_wallet_ui/constants/locale.ts

```diff
@@ -130,6 +130,9 @@ const locale = {
   backupButton: 'Backup',
   dismissButton: 'Dismiss',
 
+  // Default Wallet Banner
+  defaultWalletBanner: 'Brave Wallet is not currently set as your default web 3 provider. Change default wallet.',
```

Comment:
```suggestion
  defaultWalletBanner: 'Brave Wallet is not currently set as your default web3 provider. Change default wallet.',
```

Reply:
Will have a this string fixed shortly in this issue https://github.com/brave/brave-browser/issues/17911